src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -227,6 +227,12 @@ ov::frontend::InputModel::Ptr FrontEnd::load_impl(const std::vector<ov::Any>& va
         // this is used for OpenVINO with TensorFlow Integration
         auto graph_iterator = variants[0].as<GraphIterator::Ptr>();
         return std::make_shared<InputModel>(graph_iterator, m_telemetry);
+    } else if (variants[0].is<std::shared_ptr<IGraphIterator>>()) {
```

Comment:
should we have the same in `support_impl`?

Reply:
Added this check to support_impl(), now it is possible to run following code:
```
model = GraphIteratorTFGraph(<some tf.Graph>)
fem = FrontEndManager()
fe = fem.load_by_model(model) # returns TF frontend
```